.github/workflows/build_and_test.yml

```diff
@@ -242,8 +242,9 @@ jobs:
         key: pyspark-coursier-${{ hashFiles('**/pom.xml', '**/plugins.sbt') }}
         restore-keys: |
           pyspark-coursier-
-    - name: List Python packages (Python 3.9)
+    - name: Install Python packages (Python 3.9)
       run: |
+        python3.9 -m pip install 'mlflow>=1.0' sklearn
```

Comment:
Let's file a JIRA for the image to have `'mlflow>=1.0' sklearn`. Once the image has both packages, we can remove here. Add a comment `# TODO(SPARK-XXXXX): Install blah blah`.

Reply:
Sounds good. Just created a [ticket](https://issues.apache.org/jira/browse/SPARK-36345) and added comment.